python/semantic_kernel/schema/kernel_json_schema_builder.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import Any, Type, get_type_hints
+
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+
+
+class KernelJsonSchemaBuilder:
+
+    TYPE_MAPPING = {
```

Comment:
This should be in the file, not the class, each instance wil have their own now, which takes needless space